pkg/products/monitoring/dashboards/criticalSLOAlerts.go

```diff
@@ -4181,8 +4212,9 @@ const MonitoringGrafanaDBCriticalSLOAlertsJSON = `{
 			}
 		]
 	},
+	"refresh": "10s",
```

Comment:
Aforementioned default refresh, previously the dashboard wouldn't update automatically, now it does so every 10 seconds, looking for opinions on this time, should it be lower/higher?

Reply:
Was not having a `refresh` value set causing issues/problems? Or is this just more of an enhancement?

Reply:
It wasn't causing problems with the metrics themselves right away, it just meant that the dashboard wouldn't be updated unless the refresh button was clicked, so this is a 50/50 bug fix/enhancement, most of the other dashboards have a default automatic refresh, so I think we're probably best off having an automatic refresh for this one as well, seeing as this is a what I would consider "vital" dashboard we should be getting an update quite often. This is also something that people might not be aware of when testing E04 during RC testing, I usually refreshed mine manually and wasn't aware that there was an option to set an automatic refresh until I looked into the JSON.

Reply:
Agree, sounds reasonable. At the minimum it reduces the need to refresh to update the info.